public/static/docs/command-reference/import-url.md

```diff
@@ -48,15 +47,20 @@ determine whether the local copy is out of date.
 
 DVC supports several types of (local or) remote locations (protocols):
 
-| Type     | Discussion                                              | URL format                                 |
-| -------- | ------------------------------------------------------- | ------------------------------------------ |
-| `local`  | Local path                                              | `/path/to/local/file`                      |
-| `s3`     | Amazon S3                                               | `s3://mybucket/data.csv`                   |
-| `gs`     | Google Storage                                          | `gs://mybucket/data.csv`                   |
-| `ssh`    | SSH server                                              | `ssh://user@example.com:/path/to/data.csv` |
-| `hdfs`   | HDFS                                                    | `hdfs://user@example.com/path/to/data.csv` |
-| `http`   | HTTP to file with _strong ETag_ (see explanation below) | `https://example.com/path/to/data.csv`     |
-| `remote` | Remote path (see explanation below)                     | `remote://myremote/path/to/file`           |
+| Type     | Description                                           | `url` format                               |
+| -------- | ----------------------------------------------------- | ------------------------------------------ |
+| `local`  | Local path                                            | `/path/to/local/data`                      |
+| `s3`     | Amazon S3                                             | `s3://mybucket/data`                       |
+| `gs`     | Google Storage                                        | `gs://mybucket/data`                       |
+| `ssh`    | SSH server                                            | `ssh://user@example.com:/path/to/data`     |
+| `hdfs`   | HDFS to file\*                                        | `hdfs://user@example.com/path/to/data.csv` |
+| `http`   | HTTP to file with _strong ETag_ (explanation below)\* | `https://example.com/path/to/data.csv`     |
+| `remote` | Remote path (see explanation below)                   | `remote://myremote/path/to/data`           |
+
+> \* HDFS and HTTP **do not** support downloading entire directories, only
```

Comment:
same re HDFS

Reply:
Hmmm. Actually this is probably not true. I was just assuming because of the URL given above `hdfs://user@example.com/path/to/data.csv` which ends with a file extension, same as the HTTP example. I'll double check on this though, but I don't see a reason why we can't support HDFS dirs. Probably reverting, thanks!

Reply:
I checked internally, @efiop confirmed indeed we do NOT support HDFS directories for these commands. In fact only local, s3, gs, ssh support dirs! Will update these refs.

Reply:
Same here: No update needed! Resolving